src/vm/i386/asmhelpers.S

```diff
@@ -815,6 +815,13 @@ PATCH_LABEL ExternalMethodFixupPatchLabel
 NESTED_END ExternalMethodFixupStub, _TEXT
 
 #ifdef FEATURE_READYTORUN
+NESTED_ENTRY DynamicHelperArgsStub, _TEXT, NoHandler
+    CHECK_STACK_ALIGNMENT
+    call  eax
+    add   esp, 12
+    ret
+NESTED_END DynamicHelperArgsStub, _TEXT
```

Comment:
As I remember, ``CORINFO_HELP_READYTORUN_DELEGATE_CTOR`` in the following code invokes the stub of interest:
```
G_M65422_IG01:
       55           push     ebp
       8BEC         mov      ebp, esp
       57           push     edi
       56           push     esi

G_M65422_IG02:
       FF1500000000 call     [CORINFO_HELP_READYTORUN_NEW]
       8BF0         mov      esi, eax
       8BCE         mov      ecx, esi
       33D2         xor      edx, edx
       FF1500000000 call     [CORINFO_HELP_READYTORUN_DELEGATE_CTOR]
```
Is there any detailed description on how readytorun works? We need to change some readytorun ABI in order to implement the suggested approach (``DynamicHelperFixup``  should update caller's frame directly.  Am I right?)